src/core/worker.c

```diff
@@ -665,7 +667,7 @@ QuicWorkerLoop(
     QUIC_CONNECTION* Connection = QuicWorkerGetNextConnection(Worker);
     if (Connection != NULL) {
         QuicWorkerProcessConnection(Worker, Connection, State->ThreadID, &State->TimeNow);
-        Worker->ExecutionContext.Ready = TRUE;
+        InterlockedFetchAndSetBoolean(&Worker->ExecutionContext.Ready);
```

Comment:
Not sure these changes are strictly necessary, since they are happening **on** the worker thread (always).